src/visitpy/visit_utils/src/builtin/writescript.py

```diff
@@ -393,6 +397,11 @@ def write_window(f, prefix):
 
         set_annotation_objects(f, prefix)
 
+    # If we were given a string, open that string as a file
+    if isinstance(f, str):
+        f = open(f, 'wt')
+        WriteScript.mustCloseFile = True
```

Comment:
can this just be a normal var that you use to track this vs adding to the method? (I think its equiv but simpler?)

Reply:
sure. done. manually tested both scenarios and diffed the results and get the same result.